dev-packages/node-integration-tests/package.json

```diff
@@ -60,6 +60,7 @@
     "redis-4": "npm:redis@^4.6.14",
     "reflect-metadata": "0.2.1",
     "rxjs": "^7.8.1",
+    "tedious": "^18.6.1",
```

Comment:
`18.6.1` supports `node >= 18` [only](https://github.com/tediousjs/tedious/releases/tag/v18.6.1). 
Change to `15.1.3` to support node 14, 16, and 18.